### PR TITLE
[RHPAM-859] - Include RHPAM templates in OpenShift release

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: Filippe Spolti <fspolti@redhat.com>

### CODE REVIEW

File: official.yaml [hunk 1]
```diff
@@ -500,6 +501,34 @@ data:
         tags:
           - online-starter
           - online-professional
+  rhpam:
+    templates:
+      - location: https://raw.githubusercontent.com/jboss-container-images/rhpam-7-openshift-image/{rhpam_version}/templates/rhpam70-authoring.yaml
+        docs: https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/{rhpam_version}/templates/docs/rhpam70-authoring.adoc
+        tags:
+          - online-professional
```
Comment (bparees) [inline official.yaml]: is there a reason this specific template is being skipped for starter?

**Comment by bparees** [comment]
@sspeiche @luciddreamz ptal/signoff

**Comment by spolti** [comment]
@bparees yes, the first one requires 4gb of ram and starter has only 2 available, right?

File: official.yaml (hunk 1 above)
Comment (spolti) [reply official.yaml]: @bparees yes, the first one requires 4gb of ram and starter has only 2 available, right?

Comment (bparees) [reply official.yaml]: thanks, makes sense.

**Comment by luciddreamz** [comment]
@spolti The total memory available to end-user apps in OpenShift Online Starter is 1GiB. The second template I looked at, which has been tagged for Starter, looks like it's going to consume 1GiB (kieserver) + 512MiB (MySQL): https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/7.0.0.GA/templates/rhpam70-kieserver-mysql.yaml

@spolti And just to confirm, you followed the process and guidelines mentioned in this doc (including validation)? https://docs.google.com/document/d/1jc0EqfbcX6584taNdvSm9wio64hh6N46oVj5yUlbLoc/edit#heading=h.1594lz1p1w4s

**Comment by spolti** [comment]
@luciddreamz so, basically, I can not address none of those templates to starter tier, only pro right?

I followed this document the first time I did this process for rhdm, did I miss something?

**Comment by luciddreamz** [comment]
@spolti correct - remove the image stream and template tagging for Starter. I was just asking if you followed the doc to make sure you've tried deploying these to *some* hosted OpenShift environment :)

File: official.yaml [hunk 2]
```diff
@@ -500,6 +501,31 @@ data:
...
+      - location: https://raw.githubusercontent.com/jboss-container-images/rhpam-7-openshift-image/{rhpam_version}/templates/rhpam70-prod-immutable-kieserver.yaml
+        docs: https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/{rhpam_version}/templates/docs/rhpam70-prod-immutable-kieserver.adoc
+        tags:
+          - online-professional
+    imagestreams:
+      - location: https://raw.githubusercontent.com/jboss-container-images/rhpam-7-openshift-image/rhpam70-dev/rhpam70-image-streams.yaml
+        docs: https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.0/html/deploying_a_red_hat_process_automation_manager_7.0_authoring_environment_on_red_hat_openshift_container_platform/index
+        suffix: rhel7
+        tags:
+          - online-starter
```
Comment (bparees) [inline official.yaml]: this one still fits in starter?

Comment (spolti) [reply official.yaml]: it should be removed, thanks.

**Comment by bparees** [comment]
needs a rebase and then a fresh make import and then travis should pass and i'll merge it.

**Comment by spolti** [comment]
@bparees can this be merged as the issues were fixed?

**Pull request merged**